tutorials/ranking/init.ipynb

```diff
@@ -99,7 +99,7 @@
    "name": "python",
    "nbconvert_exporter": "python",
    "pygments_lexer": "ipython3",
-   "version": "3.6.8"
+   "version": "3.7.3"
```

Comment:
You should not change `init.ipynb` file. 
In addition, we plan to delete `init.ipynb` in future version, so you can write all your need in `dynamic_clip.ipynb`.

Reply:
init.ipynb and dynamic_clip.py have been updated.
Dynamic_clip.ipynb was already rewritten accoding to the lattest version of tuturial so i didn't change it.